tools-local/prebuilt-baseline-offline.xml

```diff
@@ -70,6 +69,7 @@
     <PackageIdentity Id="Microsoft.VisualStudio.Services.Client" Version="16.138.0-preview" />
     <PackageIdentity Id="Microsoft.Win32.Registry" Version="4.4.0" />
     <PackageIdentity Id="Newtonsoft.Json" Version="11.0.2" />
+    <PackageIdentity Id="Newtonsoft.Json" Version="12.0.2" />
```

Comment:
I wonder who's including this.  It'd be good if we could get everyone to include the version we build.

Reply:
Checking on this.

Reply:
This came from the Darc update - logged https://github.com/dotnet/source-build/issues/1040 to address.